src/components/v1/V1Project/Rewards/index.tsx

```diff
@@ -75,21 +65,16 @@ export default function Rewards() {
     : false
 
   const hasIssueTicketsPermission = useHasPermission(OperatorPermission.Issue)
-  const hasPrintPreminePermission = useHasPermission(
-    OperatorPermission.PrintTickets,
-  )
-
-  const mintingTokensIsAllowed =
-    metadata &&
-    (metadata.version === 0
-      ? canPrintPreminedV1Tickets
-      : metadata.ticketPrintingIsAllowed)
 
   const labelStyle: CSSProperties = {
     width: 128,
   }
 
-  const tokensLabel = tokenSymbol ? tokenSymbol + ' ' + t`ERC20` : t`Tokens`
+  const tokensLabel = tokenSymbolText({
```

Comment:
This change means we don't show the `ERC20` text. Not sure how important it is to include it...

Reply:
Yeah I don't think we need it